styleguide/derek/_partials/_floor.ejs

```diff
@@ -0,0 +1,4 @@
+<%- partial("_backtotop") %>
+<%- partial("_rug") %>
+<%- partial("_footer") %>
+<%- partial("_basement") %>
```

Comment:
While this is all well in good in theory, this does not seem like a good time for this. Namely because we're restructuring everything and it's probably going to conflict with the new pathing. At the very least, be prepared for a merge conflict here.